pkg/volume/glusterfs/glusterfs.go

```diff
@@ -207,6 +207,11 @@ func (b *glusterfsMounter) GetAttributes() volume.Attributes {
 	}
 }
 
+func (b *glusterfsMounter) CanMount() bool {
+	return true
```

Comment:
Can you also implement the check for Gluster. Check with @jingxu97 what the required client binaries are.